### ISSUE TITLE
Document luajit-gdb.py

### ISSUE THREAD

**Issue by rosik** [tarantool/doc#1982]
The [luajit-gdb.py](https://github.com/tarantool/luajit/blob/tarantool/src/luajit-gdb.py) isn't mentioned in doc. That's no good.

**SME:** @igormunkin

**Comment by veod32** [comment]
@igormunkin
In [2.4.1 RN](https://github.com/tarantool/tarantool/releases/tag/2.4.1), the dev issue for `luajit-gdb.py` is not mentioned. Do we have any?

**Comment by igormunkin** [comment]
> @igormunkin
> In [2.4.1 RN](https://github.com/tarantool/tarantool/releases/tag/2.4.1), the dev issue for `luajit-gdb.py` is not mentioned. Do we have any?

No, we don't. Initially, it was an "by-product" of the coredump investigation.

**Comment by veod32** [comment]
> @veod32, hope [this cover letter](https://lists.tarantool.org/tarantool-patches/cover.1580917791.git.imun@tarantool.org/) will be helpful.

Yes, it will, thanks! It's OK for the initial input. We'll get back to you later for this doc issue estimate.
